opm/parser/eclipse/EclipseState/Aquifetp.hpp

```diff
@@ -49,6 +49,18 @@ class Aquifetp {
                 V0, // initial volume of water in aquifer
                 d0; // aquifer datum depth
         std::pair<bool, double> p0;
+
+        template<class Serializer>
+        void serializeOp(Serializer& serializer)
+        {
+            serializer(aquiferID);
+            serializer(pvttableID);
+            serializer(J);
+            serializer(C_t);
+            serializer(V0);
+            serializer(d0);
+            serializer(p0);
```

Comment:
I guess that C++ would intrepet this one pod and this could be simplified in the long run (at least if we assume that all machines have the same endianess) with `serializer.pod(*this)`. But first let us get this over with.

Reply:
while true in other places, this one is not due to pair.